src/gui/MainWindow.cpp

```diff
@@ -325,7 +325,7 @@ void MainWindow::finalize()
 					Qt::CTRL + Qt::Key_M );
 
 // Prevent dangling separator at end of menu per https://bugreports.qt.io/browse/QTBUG-40071
-#if !(defined(LMMS_BUILD_APPLE) && (QT_VERSION < 0x050600))
+#ifndef LMMS_BUILD_APPLE
```

Comment:
I have small doubt about this one line tho.

Reply:
This check can be removed entirely. `QT_VERSION < 0x050600` is always false now, so `defined(LMMS_BUILD_APPLE) && (QT_VERSION < 0x050600)` is always false too, and `!(defined(LMMS_BUILD_APPLE) && (QT_VERSION < 0x050600))` is always true.

Reply:
Ok

Reply:
Removed the check.